Makefile

```diff
@@ -0,0 +1,53 @@
+.DEFAULT_GOAL := help
+
+PROJECT_NAME := regista
+IMAGE_NAME := scout
+
+NEO4J_CONTAINER_NAME := neo4j-"${PROJECT_NAME}"
+NEO4J_HTTP_PORT := 7474
```

Comment:
Maybe 7475, and 7688 (or something like this)
in the case we have a local instance for neo4j running